universal-application-tool-0.0.1/app/auth/saml/SamlCiviFormProfileAdapter.java

```diff
@@ -166,8 +166,10 @@ public CiviFormProfileData mergeCiviFormProfile(
   private String extractAttributeFromArrayList(SAML2Profile profile, String attr) {
     ArrayList attributeArray = profile.getAttribute(attr, ArrayList.class);
     StringJoiner sj = new StringJoiner(" ");
-    for (Object s : attributeArray) {
-      sj.add((String) s);
+    if (attributeArray != null) {
```

Comment:
Do the null check right after you get this on line 1 of the method; then `return "";` early if it's null.

Reply:
done